src/coreclr/jit/lsra.cpp

```diff
@@ -7448,6 +7475,11 @@ regNumber LinearScan::getTempRegForResolution(BasicBlock* fromBlock, BasicBlock*
     }
     else
     {
+        if ((freeRegs & RBM_CALLEE_TRASH) != 0)
```

Comment:
can you add a comment saying we prefer `callee-trash` registers among the available free temp registers to avoid prolog/epilog save/restore.

Reply:
Done.